src/App.jsx

```diff
@@ -24,22 +25,13 @@ export const alex = {
 };
 
 export const App = () => (
+
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+
+    <Person person={misha} />
```

Comment:
```suggestion
    <Person person={misha} />
```
Don't leave empty lines between parent and child elements